app/docs/0.10.x/plugin-development/custom-logic.md

```diff
@@ -85,6 +87,14 @@ function CustomHandler:certificate(config)
   -- Implement any custom logic here
 end
 
+function CustomHandler:rewrite(config)
```

Comment:
btw the `CHANGELOG.md` file has nothing to update... it is not mentioned in there